core-java-modules/core-java-string-algorithms-3/src/main/java/com/baeldung/emailvalidation/EmailValidation.java

```diff
@@ -0,0 +1,105 @@
+package com.baeldung.emailvalidation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.validator.routines.EmailValidator;
+
+public class EmailValidation {
+
+    public static boolean usingEmailValidator(String emailAddress) {
+
+        EmailValidator validator = EmailValidator.getInstance();
+
+        return validator.isValid(emailAddress);
+    }
+
+    public static boolean usingSimpleRegEx(String emailAddress) {
+
+        String regExPattern = "^(.+)@(\\S+)$";

```

Comment:
Can we remove all these empty lines in the methods. Maybe one after the regex string is OK.